kubernetes/services/ignite_alcor.yaml

```diff
@@ -85,9 +85,9 @@ spec:
         - name: ignite-alcor-node
           image: ignite_alcor:lib8
           imagePullPolicy: IfNotPresent
-          resources:
-            requests:
-              cpu: "16"
+          #resources:
+            #requests:
+              #cpu: "16"
```

Comment:
A pending item for us to figure out what is the recommended number of cpu. :-)

Reply:
Yes that's right